src/Components/Publishing/Layouts/__tests__/SeriesLayout.test.tsx

```diff
@@ -20,50 +20,43 @@ function mountComponent() {
   return mount(<SeriesLayout {...props} />)
 }
 
-describe("when ads are not enabled", () => {
+describe("series layout", () => {
   beforeEach(() => {
     props = {
       article: SeriesArticle,
       relatedArticles: [VideoArticle, StandardArticle],
     }
   })
 
-  it("renders a series properly", () => {
+  it("renders a series", () => {
     const series = renderComponent().toJSON()
 
     expect(series).toMatchSnapshot()
   })
 
-  it("renders a sponsored series properly", () => {
+  it("renders a sponsored series", () => {
     props.article = SeriesArticleSponsored
 
     const series = renderComponent().toJSON()
 
     expect(series).toMatchSnapshot()
   })
-
-  it("does not render a DisplayAd", () => {
-    const component = mountComponent()
-
-    expect(component.find(DisplayAd).length).toBe(0)
-  })
 })
 
-describe("when ads are enabled", () => {
+describe("series layout with ads", () => {
```

Comment:
Since there is really no concept of "with ads" vs "without ads" anymore, I think we should consolidate this describe with the one above it.